indico/modules/events/papers/templates/_contributions.html

```diff
@@ -166,3 +166,68 @@ <h2>{% trans %}Editing{% endtrans %}</h2>
         </div>
     {% endif %}
 {% endmacro %}
+
+{% macro _render_file_type_section(latest_revision_files, file_type) %}
+    <div class="files field-separator">
+        <div class="attachments-box">
+            <div class="attachments-box-header">
+                <h3 class="file-type-header">{{ file_type.name }}</h3>
+            </div>
+            <table class="tree">
+                {% for rev_file in latest_revision_files %}
+                    {% set file = rev_file.file %}
+                    {% set download_url = rev_file.download_url %}
+                    {% set icon = icon_from_mimetype(file.content_type, 'icon-file-filled') %}
+                    <tr>
+                        <td class="revision-file-row">
+                            <a href="{{ download_url }}" class="attachment {{ icon }}" target="_blank">
+                                {{ file.filename }}
+                            </a>
+                        </td>
+                    </tr>
+                {% endfor %}
+            </table>
+        </div>
+    </div>
+{% endmacro %}
+
+
+{% macro render_proceedings_section(contribution) %}
+    {% set enabled_editables = contribution.enabled_editables %}
```

Comment:
you only use this once; i don't think there's much benefit in this assignment.

BTW: using methods instead of properties for things that trigger queries on each call might be better - that way it's clearer that it's not something you should just access multiple times instead of assigning it in case you do need to access it multiple times

Reply:
This particular one gets the information from the event settings. Does that count as triggering a query?

Reply:
no. i somehow thought we were explicitly querying something in there...